shaky/src/main/java/com/linkedin/android/shaky/Shaky.java

```diff
@@ -133,6 +147,7 @@ boolean shouldIgnoreShake() {
     /**
      * @return true if we're not currently in a feedback flow, false otherwise.
      */
+    @VisibleForTesting
```

Comment:
It's already used by a test, and that's the only reason it's not private. The annotation should have been there from the beginning, but it wasn't, so I just added it now